website/docs/getting-started/template-walkthrough.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_position: 2
+---
+
+# Template Walkthrough
+
+The template generates several files, most notably:
+
+- `./package.json` includes `scripts` to `validate`, `synth`, and `deploy` your project.
+- `./tsconfig.json` configures the `@functionless/language-service` plugin to improve IDE feedback.
+- `./cdk.json` points CDK to the application entrypoint and requires the `functionless/register` hook to transpile project files.
+- `./src/app.ts` is the appliction entrypoint.
+
+Let's take a closer look at `./src/app.ts`:
+
+```typescript
+import { App, Stack } from "aws-cdk-lib";
+import { Function, StepFunction } from "functionless";
+
+const app = new App();
+
+const stack = new Stack(app, "MyStack");
+
+const sayFunction = new Function(
+  stack,
+  "SayFunction",
+  async (event: { message: string }) => {
+    console.log(event.message);
+    return;
+  }
+);
+
+new StepFunction(stack, "Workflow", async (event: { name: string }) => {
+  await sayFunction({ message: `Hello ${event.name}` });
+});
+```
+
+We start by importing constructs from `aws-cdk-lib` and `functionless`:
+
+```typescript
+import { App, Stack } from "aws-cdk-lib";
+import { Function, StepFunction } from "functionless";
+```
+
+Functionless exposes AWS CDK compatible constructs from its top level API.
+In this example, we're using the `Function` and `StepFunction` constructs which map to the AWS Lambda and AWS Step Functions services.
+
+Next, we initialize `app` which will be the root of our CDK construct tree:
+
+```typescript
+const app = new App();
+```
+
+CDK models infrastructure as a tree of constructs with a single instance of `App` at the root.
+To add constructs to the tree, we pass the parent construct as the first argument when instantiating a child construct.
```

Comment:
always capitalize Constructs?

Reply:
nit: this sentence has the word "construct" three times. Simplify?

Reply:
Done